packages/docs/src/components/landing/Seamless/Seamless.vue

```diff
@@ -5,7 +5,6 @@
         <h2 class="seamless__title">{{$t('landing.seamless.title')}}</h2>
```

Comment:
![image](https://user-images.githubusercontent.com/5394573/129356255-a4b8074e-745c-4110-b37a-4a7c897c67a6.png)

Can you remove shadow from this component?

Reply:
ok